packages/spelunker-api/src/lib/server.mjs

```diff
@@ -1,25 +1,27 @@
 import apolloExpress from 'apollo-server-express';
-import bodyParser from 'body-parser';
 import cors from 'cors';
 import express from 'express';
 
 import pipeline from './pipeline/index.mjs';
 import rootValue from './graph/root.mjs';
 import schema from './graph/schema/index.mjs';
 
-const { graphqlExpress, graphiqlExpress } = apolloExpress;
+const { ApolloServer } = apolloExpress;
 
 const server = express();
 
+const apollo = new ApolloServer({
+  schema,
+  rootValue,
+});
+apollo.applyMiddleware({ app: server, path: '/graphql' });
+
```

Comment:
This is great! 👍 

It seems Apollo Server 2/3 seem to roll with `CORS: *` by default, which may not be what we want. Should we perhaps turn it off and use our own middleware from below instead?

Reply:
Fixed! Good catch!

Reply:
I didn't reuse the middleware, but passing `cors.origin` into the `ApolloServer` config has the same effect:
https://www.apollographql.com/docs/apollo-server/api/apollo-server/#cors